src/ray/raylet/node_manager.cc

```diff
@@ -2999,6 +3000,22 @@ NodeManager::WorkersWithLatestSubmittedTasks() const {
   return workers;
 }
 
+void NodeManager::AsyncScheduleAndDispatchTasks() {
```

Comment:
hmm I'm assuming the `ScheduleAndDispatchTasks` happens on the same thread as the caller. In this case, does this really reduce the number of schedule and dispatch tasks calls?